lib/utils/rename.js

```diff
@@ -1,16 +0,0 @@
-'use strict'
```

Comment:
I'm grumpy about it but yeah, since this is fixing things for exactly the folks who would have had that patch, we probably shouldn't break the work around. =D